package/network/utils/xdp-tools/Makefile

```diff
@@ -0,0 +1,128 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=xdp-tools
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_GIT_URL:=github.com/xdp-project/xdp-tools
+PKG_SOURCE_DATE:=2022-06-21
+PKG_SOURCE_VERSION:=57e57a185af1ea6a8ac2f5b5869270e2b6543ad7
+PKG_HASH:=d52ff71481935e1a2fe992f5cbce73cf73be2e44be77670094c02d2b8db0928a
+PKG_ABI_VERSION:=$(call abi_version_str,$(PKG_SOURCE_DATE))
```

Comment:
I think it would be better to use the tagged versions instead of git hashes as the pkg version...

Reply:
Unfortunately the latest tagged version is too old to work with our libbpf (build fails due use of deprecated API and `-Werror`). I also tried backporting https://github.com/xdp-project/xdp-tools/commit/4358357639084c45980993ce47de2ddcb417fb8b as a patch, but it's too many changes. So I decided to go with git HEAD until a new version gets released.

Reply:
Alright, tagged v1.2.4 - please try that? :)

Reply:
Yes, that works. Thank you!